Child followup form addition

hatanvir opened the pull request:
taskId: HNPP-851
source: JIRA
cause: New Requirement
solution: Child followup forms added

hatanvir committed:
taskId: HNPP-851
source: JIRA
cause: New Requirement
solution: Child followup forms added

hatanvir committed:
taskId: HNPP-851
source: JIRA
cause: Optimization
solution: Child info and old followup functionality removed

mahmud6390 commented on opensrp-brac-hnpp/src/main/java/org/smartregister/brac/hnpp/interactor/TargetAchievementInteractor.java:
```diff
@@ -87,6 +87,16 @@ private ArrayList<TargetVsAchievementData> getInitialTargetAchievement(){
         visitTypeList.add(new TargetVsAchievementData(HnppConstants.EVENT_TYPE.CHILD_VISIT_0_6,HnppConstants.targetTypeMapping.get(HnppConstants.EVENT_TYPE.CHILD_VISIT_0_6)));
```
need to remove this

mahmud6390 commented on opensrp-brac-hnpp/src/main/java/org/smartregister/brac/hnpp/interactor/TargetAchievementInteractor.java:
```diff
@@ -87,6 +87,16 @@ private ArrayList<TargetVsAchievementData> getInitialTargetAchievement(){
         visitTypeList.add(new TargetVsAchievementData(HnppConstants.EVENT_TYPE.CHILD_VISIT_0_6,HnppConstants.targetTypeMapping.get(HnppConstants.EVENT_TYPE.CHILD_VISIT_0_6)));
         visitTypeList.add(new TargetVsAchievementData(HnppConstants.EVENT_TYPE.CHILD_VISIT_7_24,HnppConstants.targetTypeMapping.get(HnppConstants.EVENT_TYPE.CHILD_VISIT_7_24)));
         visitTypeList.add(new TargetVsAchievementData(HnppConstants.EVENT_TYPE.CHILD_VISIT_18_36,HnppConstants.targetTypeMapping.get(HnppConstants.EVENT_TYPE.CHILD_VISIT_18_36)));
```
need to remove this

mahmud6390 replied:
need to remove this

mahmud6390 commented on opensrp-brac-hnpp/src/main/java/org/smartregister/brac/hnpp/repository/TargetVsAchievementRepository.java:
```diff
@@ -139,11 +139,55 @@ private String getTargetName(String targetName, String baseEntityId) {
                 if(TextUtils.isEmpty(targetName)) targetName = HnppConstants.EVENT_TYPE.CHILD_FOLLOWUP;
+            }*/
+
+            else if(targetName.equalsIgnoreCase(HnppConstants.EVENT_TYPE.CHILD_FOLLOW_UP_0_3_MONTHS)){
```
should be switch case rather then else if

hatanvir committed:
taskId: HNPP-848
source: JIRA
cause: Form open error
solution: Form key updated, child visit removed from target

hatanvir committed:
taskId: HNPP-853
source: JIRA
cause: New requirement
solution: Pregnant Women Dietary Diversity form added
